phonebuddies/PhoneBuddyManager.py

```diff
@@ -102,29 +102,49 @@ def admin_buddy_pairs(self):
 
         return [ADMIN_EMAILS[:2]]
 
+    def send_admins_without_buddy_emails(self , buddy_pairs):
+        #extract a list of email addresses from buddy_pairs
+        buddy_emails_list=[]
+        for sublist in buddy_pairs:
+            buddy_emails_list.extend(sublist)
+        
+        self.admins=ADMIN_EMAILS
+        self.are_there_any_admins_without_buddies = False
+
+        #only send to admins without buddies
+        for email in self.admins:
+            if email not in buddy_emails_list:
+                self.are_there_any_admins_without_buddies = True
+                draft = EmailDraft.draft_admins_without_buddy_emails(email)
+                self._email_sender.send_email(draft)
 
+        return(self.are_there_any_admins_without_buddies)
+        
     def emails_sent_status(self):
         return "Yes" if self._really_send_emails else "Dry-run complete"
 
-
     def manage_phone_buddy_list(self):
         # Read in the database to check whether there's any issues
         buddy_pairs = []
         try:
             buddy_pairs = self._db_connector.get_buddy_email_pairs()
 
             # This is intentionally not accessed, but we want to get it so that we catch any issues
-            self._db_connector.get_all_buddies()
+            self._db_connector.get_all_buddies()         
         except Exception as e:
             ResultTracker.add_issue("Error fetching the database: %s" % str(e), save_traceback=True)
-
         # Check whether we should send out any emails and then do that
         try:
             process = self._db_connector.get_this_weeks_process()
-
             if process.phone_buddy_emails_sent:
                 ResultTracker.set_result("Phone buddy list already sent this week.")
             elif process.admin_email_sent:
+                #send emails to admins without buddies
+                self.send_admins_without_buddy_emails(buddy_pairs)
+                if self.are_there_any_admins_without_buddies:
+                    ResultTracker.set_result("One or more admins without buddies reminded")
+                else: 
+                    ResultTracker.set_result("All admins have buddies")
```

Comment:
I would move this to `send_admins_without_buddy_emails` and log the lines that you're setting as the result.

Reply:
I would use `logging.info` for this